### ISSUE TITLE
[Improvement][A11y] Present chips by group

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#427]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

.

### OS Version

.

### App version

.

### Describe the issue

Many controls are used by group and are related to the same setting or feature, for example a group of checkboxes, group of switches, group of chips, group of radios

### Expected Behavior

For each selectable grouped controls, present them with at least 2 grouped components
If possible add a visible label, used to construct the accessible name. 
The group should be labelled in a way that we understand the objective of the group, for example "filter by type of device" (for filter chips) or "select an option" (for radios)
